strapi/strapi
# Remove out of date lerna references from docs

### What does it do?

Removes a page that was added to handle an error with lerna that is no longer necessary now that we use nx

Removes a reference to lerna in the linking docs

### Why is it needed?

-

### How to test it?

Make sure the commands referenced (`yarn add some-package` and `npm install some-package` within a submodule, `yarn clean` after linking) actually work fine now

### Related issue(s)/PR(s)

-
